services/src/ogc/wms/request.rs

```diff
@@ -48,7 +51,7 @@ pub struct GetMap {
     #[serde(alias = "LAYERS")]
     pub layers: String,
     #[serde(alias = "CRS")]
-    pub crs: String, // TODO: parse CRS
+    pub crs: Option<SpatialReference>,
```

Comment:
Doesn't `SpatialReferenceOption` work?
We could use no sref if it works instead of a default?

Reply:
yeah well it was like this in wfs so i just adapted wms. I can change it if you prefer it.